doc/auth.md

```diff
@@ -230,6 +230,194 @@ Response Payload :
 ### Endpoint
 POST /forgot-password
 
+For recycle your password, you need to input your username and email for sending an password recovery to your email.
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+
+### Request Payloads
+Name | Type | Example Value
+--- | --- | ---
+username | string | jacky  
+email | string | jacky@gmail.com
+```
+{
+    "username": "jacky",
+    "email": "jacky@gmail.com"
+}
+```
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+400 | Bad Request | Bad request payload  
+404 | Not Found | User not found in database  
+500 | Internal Server Error | some un-handle error in server 
+200 | Password recovery was sended to your email. please checked your email. | Password recovery was sended to your email. please checked your email.
+```
+{
+    "status_code": "CDC-400",
+    "status_message": "Bad Request",
+    "data": null
+}
+```
+
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "Password recovery was sended to your email. please checked your email. ",
+    "data": null
+}
+```
+
+### Logic
+*if any special logic, please write down the logic here. thanks*
```

Comment:
1. Validate email input
2. Store the link forgot password to forgot_passwords table.
3. Send email reset password link to user